.github/workflows/test_rename.yml

```
@@ -0,0 +1,45 @@
+name: Rename Script CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+concurrency:
+  group: ${{ github.ref_name }}-${{ github.workflow }}
+  cancel-in-progress: true
+
+jobs:
+  rename_test:
+    name: Rename Script Test
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v2
```

v2 is very old. I think v4 is the latest
```suggestion
      uses: actions/checkout@v4
```